MicroProfile-Fault-Tolerance/tck-runner/src/test/resources/tck-suite2.0-stable.xml

```diff
@@ -10,6 +10,12 @@
         <!-- excludes -->
         <classes>
             <!-- Unstable -->
+            <class name="org.eclipse.microprofile.fault.tolerance.tck.TimeoutTest">
+                <methods>
+                    <exclude name="testLTDefaultTimeout"/>
```

Comment:
I note this is self closing where as the rest of the file isn't. Testing will confirm whether this is valid.

Reply:
Will run a test  against the branch to verify